server/consumer.go

```diff
@@ -119,9 +119,10 @@ type SequenceInfo struct {
 }
 
 type CreateConsumerRequest struct {
-	Stream string         `json:"stream_name"`
-	Config ConsumerConfig `json:"config"`
-	Action ConsumerAction `json:"action"`
+	Stream   string         `json:"stream_name"`
+	Config   ConsumerConfig `json:"config"`
+	Action   ConsumerAction `json:"action"`
+	Pedantic bool           `json:"pedantic,omitempty"`
```

Comment:
Feels like this should be part of config vs request no?

Reply:
as discussed - pedantic mode is scoped to request, not a consumer/stream.